### ISSUE TITLE
Shipping Labels: update the font and line spacing for the label refund request row

### ISSUE THREAD

**Issue by designsimply** [woocommerce/woocommerce-ios#3576]
<blockquote>[When a shipping label is refunded] can we update the font of the label refund request row header to Body? We can make it multiple lines if need be.</blockquote>
<sup>(internal reference: p5T066-1K4-p2#comment-6955 ht @Garance91540)</sup><br><br>

<table >
	<tbody>
		<tr>
			<td>Single Line</td>
			<td>Multiple Line</td>
		</tr>
		<tr>
			<td style="vertical-align:baseline;"><img width="320" alt="Screen Shot 2021-02-01 at 15 09 45" src="https://user-images.githubusercontent.com/1119271/106669102-6460c600-6568-11eb-88b1-8b32363ec86c.png"></td>
			<td style="vertical-align:baseline;"><img width="320" alt="Screen Shot 2021-02-01 at 15 12 14" src="https://user-images.githubusercontent.com/1119271/106669101-63c82f80-6568-11eb-8c79-1b5d568c7480.png"></td>
		</tr>
		<tr>
			<td colspan="2">Tested with WCiOS 5.9.0.0.</sup></td>
		</tr>
	</tbody>
</table>

**Comment by momo-ozawa** [comment]
@designsimply (cc: @Garance91540) 

I tested this on WCiOS 5.9.0.1 (iPhone SE, iOS 14.3) and it seems like the shipping label refund request row is being displayed as a multiline label in a "body" font.

https://github.com/woocommerce/woocommerce-ios/blob/eec70c4cd34f91cc8f3e52e470e2d2878fa15dcc/WooCommerce/Classes/ViewRelated/Products/Edit%20Product/Cells/ImageAndTitleAndTextTableViewCell.swift#L248-L249

<img src="https://user-images.githubusercontent.com/6711616/107326013-16127080-6aee-11eb-9054-64bedc017ad3.png" width=50%>

**Comment by jaclync** [comment]
I just tested on a few different orders, and I was able to reproduce the small font with an order that has a bunch of non-refunded and refunded labels. It looks like it's from `ImageAndTitleAndTextTableViewCell` being reused in multiple rows on this screen, like the printing info row:

https://github.com/woocommerce/woocommerce-ios/blob/7cbce8b0a4394e3b79b9e5bcd49ac1efbf03055a/WooCommerce/Classes/ViewModels/Order%20Details/OrderDetailsDataSource.swift#L466

`ImageAndTitleAndTextTableViewCell` still has a mix of ways to configure the cell (it's a tech debt item https://github.com/woocommerce/woocommerce-ios/issues/3419). In the old way configuring with a view model (`updateUI(viewModel:)`), it doesn't configure `titleLabel` to body style again so it stays the previous footnote style.

I think we can fix it by calling `titleLabel.applyBodyStyle()` before its `textColor` is set?

https://github.com/woocommerce/woocommerce-ios/blob/7cbce8b0a4394e3b79b9e5bcd49ac1efbf03055a/WooCommerce/Classes/ViewRelated/Products/Edit%20Product/Cells/ImageAndTitleAndTextTableViewCell.swift#L132-L135
